File: Python/bubble_sort.py
class BubbleSort:
    def __init__(self):
        self

    def sort(self, numbers: list[int]) -> list[int]:
        has_swapped = True
        while (has_swapped):
            has_swapped = False
            for i in range(len(numbers) - 1):
                if numbers[i] > numbers[i + 1]:
                    temp = numbers[i + 1]
                    numbers[i + 1] = numbers[i]
                    numbers[i] = temp
                    has_swapped = True
        return numbers

File: BubbleSort/src/BubbleSort.java

public class BubbleSort {

    public int[] bubbleSort(int[] numbers) {
        if (numbers.length <= 1) {
            System.out.println("Method execution aborted!");
            return numbers;
        } else {
            boolean hasSwapped;
            do {
                hasSwapped = false;
                for (int i = 0; i < numbers.length - 1; i++) {
                    if (numbers[i] > numbers[i + 1]) {
                        int temp = numbers[i];
                        numbers[i] = numbers[i + 1];
                        numbers[i + 1] = temp;
                        hasSwapped = true;
                    }
                }
            } while (hasSwapped);

            return numbers;
        }
    }
}
